src/Controls/src/BindingSourceGen/Setter.cs

```diff
@@ -4,6 +4,7 @@ public sealed record Setter(string[] PatternMatchingExpressions, string Assignme
 {
     public static Setter From(
         IEnumerable<IPathPart> path,
+        uint id,
```

Comment:
It's not clear what kind of `id` this is at a first glance. I suggest renaming this to `bindingId` to make it slightly less confusing.